
These timestamps warnings are well-intended but totally unhelpful in reality.

diff --git a/fs/namespace.c~ b/fs/namespace.c
index a6675c2..19e2cd4 100644
--- a/fs/namespace.c~
+++ b/fs/namespace.c
@@ -2855,8 +2855,6 @@ static int do_reconfigure_mnt(struct path *path, unsigned int mnt_flags)
 	unlock_mount_hash();
 	up_read(&sb->s_umount);
 
-	mnt_warn_timestamp_expiry(path, &mnt->mnt);
-
 	return ret;
 }
 
@@ -2907,8 +2905,6 @@ static int do_remount(struct path *path, int ms_flags, int sb_flags,
 		up_write(&sb->s_umount);
 	}
 
-	mnt_warn_timestamp_expiry(path, &mnt->mnt);
-
 	put_fs_context(fc);
 	return err;
 }
@@ -3292,8 +3288,6 @@ static int do_new_mount_fc(struct fs_context *fc, struct path *mountpoint,
 	if (IS_ERR(mnt))
 		return PTR_ERR(mnt);
 
-	mnt_warn_timestamp_expiry(mountpoint, mnt);
-
 	mp = lock_mount(mountpoint);
 	if (IS_ERR(mp)) {
 		mntput(mnt);
